api/user/index.spec.ts

```diff
@@ -5,7 +5,7 @@ import 'mocha'
 import * as sinon from 'sinon'
 import * as sinonChai from 'sinon-chai'
 import { mockReq, mockRes } from 'sinon-express-mock'
-import { getUserDetails } from './index'
+import { getUserDetails, getUserRoleAssignments } from './index'
```

Comment:
Looks like we are introducing semi-colons at the end of lines on the other files. Why not here?

Reply:
Agreed, its done randomly. I have change this now.